.cproject

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Done and did.  The Eclipse files are now in .gitignore.  I think we will have to add them as part of this Pull Request and then in a future commit remove them.